elliottlib/cli/find_builds_cli.py

```
@@ -307,49 +289,129 @@ def _find_shipped_builds(build_ids: List[Union[str, int]], brew_session: koji.Cl
+                continue   # Only honor pinned rpms if this tag is relevant to a RHEL version
+
+            # Honors pinned NVRs by "is"
+            pinned_by_is = builder.from_pinned_by_is(el_version, runtime.assembly, runtime.get_releases_config(), runtime.rpm_map)
+
+            # Builds pinned by "is" should take precedence over every build from tag
+            for component, pinned_build in pinned_by_is.items():
+                if component in component_builds and pinned_build["id"] != component_builds[component]["id"]:
+                    LOGGER.warning("Swapping stream nvr %s for pinned nvr %s...", component_builds[component]["nvr"], pinned_build["nvr"])
+            attachable_components |= pinned_by_is.keys()  # ART-managed rpms are always attachable
+            component_builds.update(pinned_by_is)  # pinned rpms take precedence over those from tags
+
+            # Honors group dependencies
+            group_deps = builder.from_group_deps(el_version, runtime.group_config, runtime.rpm_map)  # the return value doesn't include any ART managed rpms
+            # Group dependencies should take precedence over anything previously determined except those pinned by "is".
+            for component, dep_build in group_deps.items():
+                if component in component_builds and dep_build["id"] != component_builds[component]["id"]:
+                    LOGGER.warning("Swapping stream nvr %s for group dependency nvr %s...", component_builds[component]["nvr"], dep_build["nvr"])
+            component_builds.update(group_deps)
+
+        add_builds(component_builds)
+
+    # Will add all image member dependencies and RHCOS dependencies to the list of qualified builds we have found.
+    # Note if multiple unshipped NVRs for the same component and product version are present, Errata will probably only keep only 1 NVR.
+
+    all_el_versions = {isolate_el_version_in_brew_tag(tag) for tag in tag_pv_map}
+
+    # Honor image member dependencies
```

Sorry, I failed to communicate this.. There will be `standard` and `custom` assemblies. Only `standard` assemblies can be attached to errata. It is invalid for a standard assembly to have either image specific dependencies or `rhcos` specific dependencies. The reason is that we need standard releases to have consistent RPMs through ART images & rhcos. The only way for that to happen is if `dependencies` are defined at the group.dependency level.
 
These other dependency fields (image member & rhcos) are only for `custom` assemblies where we have all sorts of flexibility in what we deliver to customers for a hotfix (i.e. we are not constrained by the normal rules of the errata workflow).

If you find member specific or rhcos specific dependencies defined, you can throw an exception.

---

Thanks. I will fix that in the next push.